packages/dd-trace/src/appsec/iast/taint-tracking/rewriter-esm.mjs

```diff
@@ -3,29 +3,34 @@
 import path from 'path'
 import { URL } from 'url'
 import { getName } from '../telemetry/verbosity.js'
-import { isNotLibraryFile, isPrivateModule } from './filter.js'
+import { isLibraryFile, isPrivateModule } from './filter.js'
 import constants from './constants.js'
 
 const currentUrl = new URL(import.meta.url)
 const ddTraceDir = path.join(currentUrl.pathname, '..', '..', '..', '..', '..', '..')
 
-let port, rewriter
+let port, rewriter, iastEnabled
+
+let orchestrionConfig
 
 export async function initialize (data) {
   if (rewriter) return Promise.reject(new Error('ALREADY INITIALIZED'))
 
-  const { csiMethods, telemetryVerbosity, chainSourceMap } = data
+  const { csiMethods, telemetryVerbosity, chainSourceMap, orchestrion } = data
```

Comment:
From what I understood, orchestrion is a list of module_name and module_version. Therefore, this name is not very clear. Orchestrion could be a lot of things which make the code not really understandable.

Reply:
It's the config yaml for orchestrion. Would `orchestrionConfig` be preferable?

Reply:
I would say yes. At least we know it is a config !

Reply:
Done.